modules/images-managing-images-enabling-imagestreams-kube.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/managing-images/using-imagestreams-with-kube-resources.adoc
+
+
+[id="images-managing-images-enabling-imagestreams-kube_{context}"]
+= Enabling imagestreams with Kubernetes resources
+
+When using this feature, you can only reference imagestreams that reside in the
+same project as the resource. The imagestream reference must consist of a single
+segment value, for example `ruby:2.5`, where `ruby` is the name of an
+imagestream that has a tag named `2.5` and resides in the same project as the
+resource making the reference.
+
+[NOTE]
+====
+This feature can not be used in the "default" namespace, nor in any "openshift-"
+or "kube-" namespace.
+====
+
+.Procedure
+
+There are two ways to enable this:
+
+* Enabling imagestream resolution on a specific resource. This allows only this
+  resource to use the imagestream name in the image field.
+* Enabling image stream resolution on an imagestream. This allows all
+  resources pointing to this imagestream to use it in the image field.
+
+. Both of these operations can be done using `oc set image-lookup`. For example,
+the following command allows all resources to reference the imagestream named
+`mysql`:
++
+----
+$ oc set image-lookup mysql
+----
++
+This sets the `Imagestream.spec.lookupPolicy.local` field to true.
++
+.Imagestream with image lookup enabled
++
+----
+apiVersion: v1
+kind: ImageStream
+metadata:
+  annotations:
+    openshift.io/display-name: mysql
+  name: mysql
+  namespace: myproject
+spec:
+  lookupPolicy:
+    local: true
+----
++
+When enabled, the behavior is enabled for all tags within the imagestream.
+
+. Then you can query the imagestreams and see if the option is set:
++
+----
+$ oc set image-lookup
+----
+
+. You can also enable image lookup on a specific resource. This
+command allows the Kubernetes deployment named `mysql` to use imagestreams:
++
+----
+$ oc set image-lookup deploy/mysql
+----
++
+This sets the `alpha.image.policy.openshift.io/resolve-names` annotation
+on the deployment.
++
+.Deployment with image lookup enabled
++
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mysql
+  namespace: myproject
+spec:
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        alpha.image.policy.openshift.io/resolve-names: '*'
+    spec:
+      containers:
+      - image: mysql:latest
+        imagePullPolicy: Always
+        name: mysql
+----
+
+. To disable image lookup, pass `--enabled=false`:
++
```

Comment:
A modular docs purist might consider this a separate module, as it addresses a different user story